### ISSUE TITLE
Add 480i output modes for classic CRTs and light guns

### ISSUE THREAD

**Issue by BuckoA511** [MakeMHz/xbox-hd-plus#41]
Please add a digitisation only option that allows for 480i output, for people wanting to occasionally connect classic, 15khz only CRTs via a digital to analogue converter (DAC) such as the HD Fury 2.

The Xbox HD+ mod removes all analogue video output options, that means people wishing to occasionally use their consoles on older CRT televisions cannot do so, as disabling 480p in Xbox dashboard no longer works, and output resolution is always at least 480p.

This means that effectively this functionality is removed from the Xbox console after adding the HD+ mod, even if you are in possession of a digital to analogue converter.

Please add support for 480i so that no functionality is lost when upgrading to the HD+ or produce a kit that allows for analogue video output to be retained along side HDMI.

**Comment by BuckoA511** [comment]
Yes and that works well, but that's beside the point, classic CRT TVs do not support 480p.

**Comment by mikegreiling** [comment]
I would also very much be interested in 480i support. The built-in DAC in the OG Xbox is notoriously bad and one could imagine running the clean XboxHD+ output through a simple external HDMI->YPbPr DAC would work very well for playing OG Xbox on a 15khz CRT with lightgun support if only it supported resolutions below 480p…

It sounded like this was in the works at some point, but I have not seen anything since this old post from @LoveMHz on the MakeMHZ discord:

![IMG_8838](https://github.com/MakeMHz/xbox-hd-plus/assets/812328/d9df01a7-4795-4607-b345-8de539d3a6ea)
